.github/workflows/build-distribute.yml

```
@@ -0,0 +1,127 @@
+          export FIREBASE_APP_ID=${{ secrets.FIREBASE_APP_ID }}
+          export FIREBASE_CLI_TOKEN=${{ secrets.FIREBASE_CLI_TOKEN }}
+          export TESTER_GROUPS=${{ secrets.TESTER_GROUPS }}
+          export GOOGLE_PLAY_JSON_KEY=$(echo ${{ secrets.GOOGLE_PLAY_JSON_KEY }} | base64 --decode)
+
+          if [ "${{ inputs.build_type }}" == "debug" ]; then
+            bundle exec fastlane build_and_distribute_debug variant:${{ inputs.variant }}
+          else
+            bundle exec fastlane build_and_distribute_release variant:${{ inputs.variant }}
+          fi
+
+          # Remove the keystore file after signing 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)